src/Valet/Models/Docker/Manifest.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+
+namespace Valet.Models.Docker;
+
+public class Manifest
+{
+    public int schemaVersion { get; set; }
+    public string? mediaType { get; set; }
+    public ManifestConfig? config { get; set; }
+    public List<ManifestConfig>? layers { get; set; }
```

Comment:
```suggestion
    [JsonProperty("schemaVersion")]
    public int SchemaVersion { get; set; }

    [JsonProperty("mediaType")]
    public string? MediaType { get; set; }

    [JsonProperty("config")]
    public ManifestConfig? Config { get; set; }

    [JsonProperty("layers")]
    public List<ManifestConfig>? Layers { get; set; }
```

By convention, properties should be PascalCase'd in c#. Adding a `JsonProperty` attribute can be used to force the serialization/deserialization to use camelCase names